src/dialogs/voice-assistant-setup/voice-assistant-setup-step-pipeline.ts

```diff
@@ -117,19 +117,24 @@ export class HaVoiceAssistantSetupStepPipeline extends LitElement {
           Install add-ons or containers to run it on your own system. Powerful
           hardware is needed for fast responses.
         </p>
-        <a
-          href=${documentationUrl(
-            this.hass,
-            "/voice_control/voice_remote_local_assistant/"
-          )}
-          target="_blank"
-          rel="noreferrer noopenner"
-        >
-          <ha-button @click=${this._skip}>
-            <ha-svg-icon .path=${mdiOpenInNew} slot="icon"></ha-svg-icon>
-            Learn more</ha-button
+        <div class="row">
+          <a
+            href=${documentationUrl(
+              this.hass,
+              "/voice_control/voice_remote_local_assistant/"
+            )}
+            target="_blank"
+            rel="noreferrer noopenner"
           >
-        </a>
+            <ha-button @click=${this._skip}>
```

Comment:
```suggestion
            <ha-button>
```

I meant this button, we have a skip in top right.